nerdlandbot/translations/Translations.csv

```diff
@@ -129,3 +129,6 @@
 "purger_list_title","List of channels with automatic removal of messages and max age:","Lijst van kanalen waar berichten automatisch worden verwijderd en de maximale leeftijd:"
 "purger_list_item","  - <#{0}>: {1} day(s)","  - <#{0}>: {1} dag(en)"
 "channel_is_voice","You should provide a text channel, not a voice channel","Geef een tekst kanaal in plaats van een voice kanaal, foemp!"
+"privacy_brief","Prints the privacy policy.","Toont de privacy policy die door de bot gehanteerd wordt."
+"privacy_policy_title","Privacy Policy","Privacy Policy"
+"privacy_policy_content","This bot was developed with privacy as one of our core ideals.\nBecause of this we have formulated a few statements about the inner workings.\n\nThe bot will only listen to user commands.\nWe will not parse any user messages, nor track any reactions, unless those made specifically to interact with the bot.\nWe will not track any user data, except for your user ID, as this is required to notify you.\nWe will not store any data about your messages, reactions, or other actions you take on discord.\n\nThe logs we keep are strictly for debugging purposes, and will not contain any personal info.","Bij de ontwikkeling van deze bot hebben we privacy bekeken als een zeer belangrijk punt, om die reden hebben we onze mening hierover uitgedrukt in enkele punten.\n\nDe bot luistert enkel naar commando's die beginnen met de bepaalde prefix.\nDe bot zal nooit berichten die niet met de gegeven prefix beginnen parsen, ook huden we geen reacties bij, behalve de reacties op berichten van de bot.\nDe bot houdt geen gebruikersdata bij, behalve het gebruikers-ID, aangezien dit nodig om gebruikers te taggen.\nDe bot houdt geen data bij over je berichten, reacties, of andere acties op discord.\n\nDe logs die we bijhouden worden enkle gebruikt om te debuggen. Deze bevatten geen persoonlijke data."
```

Comment:
,"Bij de ontwikkeling van deze bot hebben we privacy bekeken als een zeer belangrijk punt, om die reden hebben we onze mening hierover uitgedrukt in enkele punten.\n\nDe bot luistert enkel naar commando's die beginnen met de bepaalde prefix.\nDe bot zal nooit berichten die niet met de gegeven prefix beginnen parsen, ook **_houden_** we geen reacties bij, behalve de reacties op berichten van de bot.\nDe bot houdt geen gebruikersdata bij, behalve het gebruikers-ID, aangezien dit nodig om gebruikers te taggen.\nDe bot houdt geen data bij over je berichten, reacties, of andere acties op discord.\n\nDe logs die we bijhouden worden **_enkel_** gebruikt om te debuggen. Deze bevatten geen persoonlijke data."

I found two typo's in the Dutch text, I have corrected them and made the words bold. (houden in the third sentence and enkel in the last)

Reply:
should be fixed